tensorflow_lite_support/python/task/processor/proto/classifications_pb2.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Classifications protobuf."""
+
+from tensorflow_lite_support.cc.task.vision.proto import classifications_pb2
```

Comment:
Same: please move to python/task/vision/proto.

Reply:
@mbrenon I think it makes more sense in `python/task/core/proto` as this will be shared between vision, text and audio.